src/assets/style/stripView.css

```diff
@@ -0,0 +1,123 @@
+.stripView {
+    height: calc(100% - 36px);
+    width: 400px;
+    position: absolute;
+    top: 0;
+    right: 0;
+    margin: 0 auto;
+    font-family: Monospace, serif;
+    font-size: 10px;
```

Comment:
my only concern is the loss in screen space.  at that size, having the strip list open will take up (to my eye) about 10% more space.  We can do it, thats easy, I just feel like that font might be _too_ big.